langchain_dappier/retrievers.py

```
@@ -0,0 +1,212 @@
+                num_articles_ref=self.num_articles_ref,
+                search_algorithm=self.search_algorithm,
+            )
+            return self._extract_documents(response=response)
+        except Exception as e:
+            raise ValueError(f"Error while retrieving documents: {e}") from e
+
+    async def _aget_relevant_documents(
+        self,
+        query: str,
+        *,
+        run_manager: AsyncCallbackManagerForRetrieverRun,
```

Ruff (0.8.2)

Unused method argument: `run_manager`

(ARG002)